Libraries/LibWeb/HTML/HTMLImageElement.cpp

```diff
@@ -411,10 +411,15 @@ Optional<ARIA::Role> HTMLImageElement::default_role() const
 {
     // https://www.w3.org/TR/html-aria/#el-img
     // https://www.w3.org/TR/html-aria/#el-img-no-alt
+    // https://w3c.github.io/aria/#image
+    // N.B. The “image” role value is a synonym for the older “img” role value; however, the el-img test in
```

Comment:
We haven't decided to use `n.b.` over `NOTE:`, so let's not employ new comment styles here yet - let's stick with `NOTE:` for now (with hanging indents if it wraps to multiple lines).

Reply:
OK — made it so